build.gradle.kts

```diff
@@ -7,7 +7,7 @@ version = "1.0.0"
 object Versions {
     const val flyway = "8.5.13"
     const val hikari = "5.0.1"
-    const val jackson = "2.13.3"
+    const val jackson = "2.13.4"
```

Comment:
Kanskje denne endringen er tilstrekkelig? (dvs dette alene vil kanskje overstyre versjonen referert fra ktor).

Reply:
Prøvde det, og hvertfall på dependency treet ser det ut som at gradle bruker den nyeste den finner alle steder dette biblioteket brukes. Prøver å pushe det ut for å se om snyk blir fornøyd med det.